### ISSUE TITLE
ntp run condor merge

### ISSUE THREAD

**Issue by DouglasBurns** [BristolTopGroup/NTupleProduction#214]
From what I gather at the moment ntp run condor waits for all jobs to finish before merging the output. For me (I don't know about others), condor seems to hang on a few jobs, especially when large number of jobs have been submitted. Could a decent time limit be placed on how long each job can run for before cutting losses and merging what has been completed?

**Comment by kreczko** [comment]
Yes, and I have been looking into possible solutions. In the current DAG setup this is not possible and I am brainstorming ways to change that. One possibility is to exchange htcondenser for Ganglia and use the Ganglia tasks. Need to do more reading.
